modules/graphql/src/sbomstatus.rs

```diff
@@ -0,0 +1,34 @@
+use actix_web::web;
+use async_graphql::{Context, FieldResult, Object};
+use std::sync::Arc;
+use trustify_common::db::{Database, Transactional};
+use trustify_common::id::Id;
+use trustify_module_fundamental::sbom::model::details::SbomStatus;
+use trustify_module_fundamental::sbom::service::SbomService;
+use uuid::Uuid;
+
+#[derive(Default)]
+pub struct SbomStatusQuery;
+
+#[Object]
+impl SbomStatusQuery {
```

Comment:
Have you a strategy for unit-testing the GraphQL aspects?

Reply:
Yes, I need to look into that.